parts/k8s/kubernetesmastervars.t

```diff
@@ -93,7 +93,7 @@
     "apiVersionNetwork": "2017-10-01",
     "environmentJSON": "{{GetCustomEnvironmentJSON}}",
 {{ else }}
-    "apiVersionCompute": "2018-06-01",
+    "apiVersionCompute": "2018-10-01",
```

Comment:
Do we know how this change interacts with the cluster autoscaler? If the VMSS was created at this API version, do all subsequent updates need to come through this API as well?

Reply:
Good catch, this change should be wrapped inside `{{if .VMSSOverProvisioningEnabled}}` for now, to be safe.